tests/integration/aclgen_test.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2015 The Capirca Project Authors All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import sys
+import logging
+import os
+import inspect
+import shutil
+import tempfile
+from cStringIO import StringIO
+import filecmp
+
+import aclgen
+from lib import policy
+from lib import naming
+
+
+class Test_AclGen_ensure_demo_files_work(unittest.TestCase):
+  """Ensure Capirca demo runs successfully out-of-the-box."""
+
+  def setUp(self):
+    self.iobuff = StringIO()
```

Comment:
Above do a similar import as in aclgen for gflags as flags.
Then between the class definition and imports add "FLAGS = flags.FLAGS" and somewhere in the setup method add FLAGS.Reset() and things should work.